docs/lua/functions-5.md

```
@@ -3627,6 +3627,34 @@ Checks if a given room is Mario's current room, even if on an object
 <br />
 
+## [trigger_obj_dialog_when_facing](#trigger_obj_dialog_when_facing)
+
+### Description
+Checks if a given room is Mario's current room, even if on an object
+
+### Lua Example
+`local integerValue = trigger_obj_dialog_when_facing(m, inDialog, dialogID, dist, actionArg, inContinueDialogFunction)`
+
+### Parameters
+| Field | Type |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)